freebsd build warning

```
/usr/home/julia/julia-fbsd-buildbot/worker/freebsdci/build/src/init.c:100:9: warning: implicit declaration of function 'pthread_attr_get_np' is invalid in C99 [-Wimplicit-function-declaration]
        pthread_attr_get_np(pthread_self(), &attr);
        ^
/usr/home/julia/julia-fbsd-buildbot/worker/freebsdci/build/src/init.c:100:9: warning: this function declaration is not a prototype [-Wstrict-prototypes]
2 warnings generated.
```
https://build.julialang.org/#/builders/68/builds/2202/steps/6/logs/stdio

--- a/src/init.c
+++ b/src/init.c
@@ -17,6 +17,10 @@
 #include <getopt.h>
 #endif
 
+#if defined(_OS_FREEBSD_)
+#include <pthread_np.h>
+#endif
+
 #include "julia.h"
 #include "julia_internal.h"
 #define DEFINE_BUILTIN_GLOBALS

